dev/storybook/src/stories/sparkling-controls/toolbar/Toolbar.svelte

```diff
@@ -2,38 +2,38 @@
 </script>
 
 <div class="crm-toolbar-toolbar">
-	<div class="left-panel">
```

Comment:
Why it is changed here?

Reply:
I just fix all file formatting, include this (replaced tab to spaces)

Reply:
you do not touched this file by your changes, so better to perform such fixes in separate PR.s